operations/customers.md

```diff
@@ -0,0 +1,169 @@
+- [Sparse fieldsets](../guidelines/sparse-fieldsets.md) - supports all fields of `customer` query parameter.
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/customers`
+
+### Response
+
+```javascript
+{
+  "data": [
+    {
+      "id": "31b14937-2524-491f-b0a0-dc0a7393ff33",
+      "type": "customers",
+      "attributes": {
+        "email": "example@email.com",
+        "fullName": "Name",
+        "taxNumber": "tax1234",
+        "dateOfBirth": "2024-10-24",
+        "phone": "+123123123",
+        "mobile": "+123123123",
+        "address1": "Address",
+        "address2": "Address",
+        "city": "City",
+        "state": "State",
+        "postalCode": "10000",
+        "country": "HR",
+        "notes": "This is a note",
+        "countrySpecificCode": "code",
+        "companyName": "Company name",
+        "createdAt": "2024-10-16T14:30:00Z",
+        "updatedAt": "2024-10-19T14:30:00Z"
+      }
+    }
+  ],
+  "links": {
+    "prev": "https://pos.mews-demo.com/api/v2/customers?page%5Bbefore%5D=NA&page%5Bsize%5D=1",
+    "next": "https://pos.mews-demo.com/api/v2/customers?page%5Bafter%5D=NA&page%5Bsize%5D=1"
+  }
+}
+```
+Below is a list of all possible fields this endpoint can return including relationships fields fetched with include query parameter.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `fullName` | string,null | optional, max length 255 characters | The full name of the customer. This can be either a personal name or a company name. |
+| `companyName` | string,null | optional, max length 255 characters | The name of the company associated with the customer. |
+| `taxNumber` | string,null | optional, max length 40 characters | The customer's tax identification number. |
+| `email` | string,null | optional, max length 320 characters | The customer's email address. |
+| `address1` | string,null | optional, max length 255 characters | The first line of the customer's address. |
+| `address2` | string,null | optional, max length 255 characters | The second line of the customer's address. |
+| `city` | string,null | optional, max length 100 characters | The city of the customer's address. |
+| `state` | string,null | optional, max length 100 characters | The state or province of the customer's address. |
+| `postalCode` | string,null | optional, max length 20 characters | The postal or ZIP code of the customer's address. |
+| `country` | string,null | optional, max length 100 characters | The country of the customer's address. |
+| `notes` | string,null | optional, max length 500 characters | Additional notes about the customer. |
+| `phone` | string,null | optional, max length 20 characters | The customer's phone number. |
+| `mobile` | string,null | optional, max length 20 characters | The customer's mobile phone number. |
+| `countrySpecificCode` | string,null | optional, max length 50 characters | A unique country specific code assigned to the customer. Lottery code for example in Italy. |
```

Comment:
I'm really curious as to what a country specific code might be, this is a good example of where it would be helpful to have a real-life JSON example (see comments above).

Reply:
It says that it is a lottery code for example in Italy. Does that help? Regardless, I will update all examples as suggested.